deprecated/bookmark+-doc.el

```diff
@@ -839,7 +839,7 @@
 ;;  (see (@> "Bookmarks for Specific Files or Buffers")).
 ;;
 ;;  All bookmark jump commands are bound to keys that have the prefix
-;;  `C-x j'.  There is an other-window version of most jump commands,
+;;  `C-x j'.  There is an `other-window` version of most jump commands,
```

Comment:
This isn't strictly a spelling fix. `an other` trips over a pattern I've added (even Chrome hates it). It seems useful to tag out `other-window`, but upon review, I'm pretty sure I used the wrong markup...